nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -775,6 +775,80 @@ def _get_convert_map(opset):
     }
 
 
+supported_ops = set([
```

Comment:
I am thinking defining the supported_ops set is redundant.
checking op_name in identity_list or op_name in convert_map or op_name = 'Constant' should be enough